packages/plugins/rubygem-redhat_access/rubygem-redhat_access.spec

```diff
@@ -0,0 +1,131 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name redhat_access
+
+%global foreman_dir /usr/share/foreman
+%global foreman_bundlerd_dir %foreman_dir/bundler.d
+#%global foreman_assets_dir %foreman_dir/public/assets
+%global rubygem_redhat_access_dir %{gem_dir}/gems/%{gem_name}-%{version}
+%global gem_spec %{gem_dir}/specifications/%{gem_name}-%{version}.gemspec
+%global gem_cache %{gem_dir}/cache/%{gem_name}-%{version}.gem
+
+
+
+%global scl_rake /usr/bin/%{?scl:%{scl_prefix}}rake
+
+Name: %{?scl_prefix}rubygem-foreman-%{gem_name}
+Version: 2.1.0
+Release: 1%{?dist}
+Summary: Foreman engine to access Red Hat knowledge base and manage support cases.
+Group: Development/Languages
+License: GPLv2+
+URL: https://github.com/redhataccess/foreman-plugin
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+
+Requires: foreman >= 1.15.0
+Requires: katello >= 3.4.0
```

Comment:
I think it's unwise to depend on the katello metapackage. It might be better to depend on the gem instead.